bands/configspec.ini

```diff
@@ -27,12 +29,11 @@ bands = string(default="Band Management")
 [[band_merch]]
 no_merch = string(default="Not selling merch")
 own_table = string(default="Dedicated table")
-rock_island = string(default="Rock Island")
```

Comment:
As it happens, there's a poorly documented ("completely undocumented" would probably be more honest) feature in the way enums work, which is that if you set an option to the empty string, then our config parsing code will remove it from the options list.

In other words, we could keep this option here, and then if we set ``rock_island = ""`` in our puppet-managed ``development.ini``, then ``c.ROCK_ISLAND`` will still exist, but the value will be absent from ``c.BAND_MERCH_OPTS``.

So this change is fine, but not actually necessary.

I should probably document some more of this stuff :/